python-checks/src/test/resources/checks/nonCallableCalledImporter.py

```diff
@@ -1,7 +1,8 @@
 from nonCallableCalledImported import MyNonCallableClass, MyCallableClass
-
+import nonCallableCalledImported
 
 imported_non_callable = MyNonCallableClass()
+imported_non_callable_alias = nonCallableCalledImported.MyNonCallableClassAlias()
 imported_callable = MyCallableClass()
 
 imported_non_callable()  # Noncompliant
```

Comment:
I discovered this FN on master : 
```suggestion
imported_non_callable()  # Noncompliant
imported_non_callable_alias()  # FN : this is non callable
```
I didn't want to create confusion in this PR so I didn't commit it.